examples/scenes/src/test_scenes.rs

```diff
@@ -1,8 +1,12 @@
+use std::sync::Arc;
+
 use crate::{ExampleScene, SceneConfig, SceneParams, SceneSet};
-use vello::kurbo::{Affine, BezPath, Ellipse, PathEl, Point, Rect};
+use vello::kurbo::{Affine, BezPath, Ellipse, PathEl, Point, Rect, Vec2};
 use vello::peniko::*;
 use vello::*;
 
+const PIET_LOGO_IMAGE: &[u8] = include_bytes!("../../assets/piet-logo.png");
```

Comment:
Not so sure about using the Piet logo here. Ideally, we'd have a Vello logo, or a Linebender logo.

I don't have a better suggestion for what we can use.

Reply:
I assume we're free to use the druid logo as well but I thought piet might be slightly more closely related to vello given the lineage. For now, the priority is just to have *some* example that displays an image.

I can provide colorful... commentary on a logo project :) I don't think my artistic talents are up to the task!